docs/monitoring-getting-started.md

```diff
@@ -48,5 +48,5 @@ Within the genestack repo we can update our alerting rules via the alerting_rule
 View alerting_rules.yaml in:
 
 ``` shell
-less /etc/genestack/kustomize/prometheus/alerting_rules.yaml
+less /etc/genestack/base-helm-configs/prometheus/alerting_rules.yaml
```

Comment:
should be /etc/genestack/helm-configs? the 'base' part seems dropped /etc/genestack

Reply:
This would be pointing to the base configs though and not the custom/regional overrides?

Reply:
eh, i suppose you're technically right here, but up to this point in the docs there's no concept of the 'config' dir that we're using for 'multi-region-support'. Unless that's changed recently?

Reply:
Looked at other examples and  made the change